ccdImports/conversionScripts/importAllXLS.sh

```diff
@@ -3,7 +3,7 @@
 set -eu
 
 binFolder=$(dirname "$0")/../../bin
-xlsToJsonFolder=$(dirname "$0")/../../jsonToXLS-"$1"
+xlsToJsonFolder=$(dirname "$0")/../../jsonToXLS
```

Comment:
"$1" do we require these parameter?

Reply:
I thought we were all having this issue, so I removed it
The build is just going thro again, so I will check any impact

Reply:
can Imports xls on local setup with this change